src/core_scan/Core_scan.ml

```diff
@@ -761,10 +767,8 @@ let extracted_targets_of_config (config : Core_scan_config.t)
                extractors xtarget
            in
            (* Print number of extra targets so Python knows *)
-           (match config.output_format with
-           | Json true when extracted_targets <> [] ->
-               pr (string_of_int (List.length extracted_targets))
-           | _ -> ());
+           if Common.null extracted_targets then
```

Comment:
BUG: if it's null we print extracted_targets?
I'm curious why our tests didn't find this bug. Maybe we don't really test extract mode.
Any way I've fixed this code in https://github.com/semgrep/semgrep/pull/9326

Reply:
Ah nice catch. We likely didn't catch since this should just affect the CLI's progress bar, which I doubt we test (let alone test for extract mode).